install.ps1

```diff
@@ -4,9 +4,16 @@
 
 $ErrorActionPreference = 'Stop'
 
+if ($v) {
+  $Version = "v${v}"
+}
+if ($args.Length -eq 1) {
+  $Version = $args.Get(0)
```

Comment:
I gave the first argument precedence over $v since it's a more direct way to provide the version number to the script.